server-phoenix/lib/helios_web/schema/helpers/location.ex

```diff
@@ -1,8 +1,4 @@
 defmodule HeliosWeb.Schema.Helpers.Location do
-  def time_now(time_zone) do
-    DateTime.now!(time_zone)
-  end
```

Comment:
This looks like a separate commit: `refactor: move time_now to location modal`. I'm also curious if this broke other calls.

Reply:
The refactor commit just deletes `Location.time_now/1`. I was expecting to see it added elsewhere and references updated.